### PR TITLE
Fixes

### LINKED ISSUE #11: README & pyproject.toml should agree on python version
These two should agree:

https://github.com/BCDA-APS/bs_model_instrument/blob/c9ee8dd34524d2776e4fb54d526329d804284c0d/pyproject.toml#L159-L160

https://github.com/BCDA-APS/bs_model_instrument/blob/c9ee8dd34524d2776e4fb54d526329d804284c0d/README.md?plain=1#L19

### PR DESCRIPTION
**Pull request by MDecarabas**
- Fixed MD_path to not cause error out
- closes #11

### CODE REVIEW

**Comment by ravescovi** [comment]
Not passing tests.

- Add better explanation on where the MD location is pointing too. Use a more common location for the standard package.
- Remove python versions from tests (https://github.com/BCDA-APS/bs_model_instrument/blob/main/.github/workflows/code.yml)

**Comment by MDecarabas** [comment]
> Not passing tests.
> 
> * Add better explanation on where the MD location is pointing too. Use a more common location for the standard package.
> * Remove python versions from tests (https://github.com/BCDA-APS/bs_model_instrument/blob/main/.github/workflows/code.yml)

Removed 3.9 & 3.10 tests. What are some suggestions for bullet point one that you would recommend? The location is highlighted in the iconfig file.

File: src/instrument/core/run_engine_init.py [hunk 1]
```diff
@@ -31,9 +31,12 @@
 
 # Save/restore RE.md dictionary, in this precise order.
 if MD_PATH is not None:
-    RE.md = bluesky.utils.PersistentDict(MD_PATH)
-RE.md.update(re_metadata(cat))  # programmatic metadata
-RE.md.update(re_config.get("DEFAULT_METADATA", {}))
+    try:
+        RE.md = bluesky.utils.PersistentDict(MD_PATH)
+        RE.md.update(re_metadata(cat))  # programmatic metadata
```
Comment (prjemian) [inline src/instrument/core/run_engine_init.py:36]: These two updates can be moved outside the `try..except..` block.

File: src/instrument/core/run_engine_init.py [hunk 2]
```diff
@@ -31,9 +31,12 @@
...
+        RE.md.update(re_config.get("DEFAULT_METADATA", {}))
+    except Exception as _:
```
Comment (prjemian) [inline src/instrument/core/run_engine_init.py:38]:
```suggestion
    except Exception:
```

File: src/instrument/core/run_engine_init.py [hunk 3]
```diff
@@ -31,9 +31,12 @@
...
+        print("\n cannot access .config folder \n")
```
Comment (prjemian) [inline src/instrument/core/run_engine_init.py:39]:
```suggestion
        print("\n Could not create PersistentDict for RE metadata.  Continuing without save that to disk.  ({reason}) \n")
```

File: src/instrument/core/run_engine_init.py [hunk 4]
```diff
@@ -32,8 +32,16 @@
 # Save/restore RE.md dictionary, in this precise order.
 if MD_PATH is not None:
     RE.md = bluesky.utils.PersistentDict(MD_PATH)
```
[the remark this answers is not in the document]
Comment (MDecarabas) [reply src/instrument/core/run_engine_init.py:34]: didn't you just say to move this outside the try, except?

**Review by ravescovi — APPROVED** [review]

File: src/instrument/core/run_engine_init.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (prjemian) [reply src/instrument/core/run_engine_init.py:34]: > These two updates can be moved outside the try..except.. block.

like this (all the way outside the `if MD_PATH block`):

```py
if MD_PATH is not None:
    try:
        RE.md = bluesky.utils.PersistentDict(MD_PATH)
    except Exception as e:
        print(
            f"\n Could not create PersistentDict for RE metadata. Continuing without "
            f"saving metadata to disk. The error is: {e} \n"
        )

RE.md.update(re_metadata(cat))  # programmatic metadata
RE.md.update(re_config.get("DEFAULT_METADATA", {}))
```

[the remark this answers is not in the document]
Comment (MDecarabas) [reply src/instrument/core/run_engine_init.py:34]: Ok, previous comment highlighted different code

**Review by prjemian — APPROVED** [review]
That's it.

**Pull request merged**